app/controllers/containers_controller.rb

```diff
@@ -16,7 +16,7 @@ def create
       if @container.errors.none?
         format.html { redirect_to cluster_path(@container.cluster), notice: 'Container was successfully created.' }
       else
-        format.html { render :new }
+        format.html { redirect_to new_container_path(cluster_id: @cluster.id), notice: 'Error when creating container.' }
```

Comment:
if we do redirect, aren't the fields error highlight will be missing?

Reply:
Forget about that thing. will fix this changes.

Reply:
Done. https://github.com/pathfinder-cm/pathfinder-mono/pull/26/commits/0cdad2ddbaba3ce0b8ecc9500963501422c81fc0